src/Components/Popover/Base/Popover.tsx

```diff
@@ -55,6 +55,8 @@ export interface PopoverProps extends StandardFunctionProps {
   triggerRef: RefObject<any>
   /** Adds reasonable styles to popover dialog contents so you do not have to */
   enableDefaultStyles?: boolean
+  /** Ensures the popover appears above all other portal elements */
+  forceToTop?: boolean
```

Comment:
We can hope this is used sparingly but I feel like once people start to use this in an application subsequent popovers will also end up having it applied. Seems like a slippery slope to all elements having this applied which would defeat the purpose. This isn't a problem for the library as much as `!important` is a problem for CSS, but maybe we should add some documentation around it to promote good practices.